src/transformers/models/clip/modeling_tf_clip.py

```diff
@@ -169,7 +169,12 @@ def build(self, input_shape: tf.TensorShape = None):
                 name="embeddings",
             )
 
-        super().build(input_shape)
+        if self.built:
```

Comment:
Same here - can we return quickly without calling `self.add_weight` if `self.built` is True?